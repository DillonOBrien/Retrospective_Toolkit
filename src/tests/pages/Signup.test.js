import React from "react";
import { shallow } from "enzyme";
import Signup from "../../pages/Signup";

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Signup />);
});

describe("Pages/Signup Testing", () => {
  // Snapshot for Signup
  test("SNAPSHOT: Signup", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
