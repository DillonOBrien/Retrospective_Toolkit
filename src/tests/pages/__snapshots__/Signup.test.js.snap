// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages/Signup Testing SNAPSHOT: Signup 1`] = `
<WithStyles(ForwardRef(Container))
  component="main"
  maxWidth="xs"
>
  <WithStyles(CssBaseline) />
  <div
    className="makeStyles-paper-1"
  >
    <WithStyles(ForwardRef(Avatar))
      className="makeStyles-avatar-2"
    >
      <Memo([object Object]) />
    </WithStyles(ForwardRef(Avatar))>
    <WithStyles(ForwardRef(Typography))
      component="h1"
      variant="h5"
    >
      Sign up
    </WithStyles(ForwardRef(Typography))>
    <form
      className="makeStyles-form-3"
      noValidate={true}
    >
      <WithStyles(ForwardRef(Grid))
        container={true}
        spacing={2}
      >
        <WithStyles(ForwardRef(Grid))
          item={true}
          sm={6}
          xs={12}
        >
          <WithStyles(ForwardRef(TextField))
            autoComplete="fname"
            autoFocus={true}
            fullWidth={true}
            id="firstName"
            label="First Name"
            name="firstName"
            required={true}
            variant="outlined"
          />
        </WithStyles(ForwardRef(Grid))>
        <WithStyles(ForwardRef(Grid))
          item={true}
          sm={6}
          xs={12}
        >
          <WithStyles(ForwardRef(TextField))
            autoComplete="lname"
            fullWidth={true}
            id="lastName"
            label="Last Name"
            name="lastName"
            required={true}
            variant="outlined"
          />
        </WithStyles(ForwardRef(Grid))>
        <WithStyles(ForwardRef(Grid))
          item={true}
          xs={12}
        >
          <WithStyles(ForwardRef(TextField))
            autoComplete="email"
            fullWidth={true}
            id="email"
            label="Email Address"
            name="email"
            required={true}
            variant="outlined"
          />
        </WithStyles(ForwardRef(Grid))>
        <WithStyles(ForwardRef(Grid))
          item={true}
          xs={12}
        >
          <WithStyles(ForwardRef(TextField))
            autoComplete="current-password"
            fullWidth={true}
            id="password"
            label="Password"
            name="password"
            required={true}
            type="password"
            variant="outlined"
          />
        </WithStyles(ForwardRef(Grid))>
        <WithStyles(ForwardRef(Grid))
          item={true}
          xs={12}
        >
          <WithStyles(ForwardRef(FormControlLabel))
            control={
              <WithStyles(ForwardRef(Checkbox))
                color="primary"
                value="allowExtraEmails"
              />
            }
            label="I want to receive inspiration, marketing promotions and updates via email."
          />
        </WithStyles(ForwardRef(Grid))>
      </WithStyles(ForwardRef(Grid))>
      <WithStyles(ForwardRef(Button))
        className="makeStyles-submit-4"
        color="primary"
        fullWidth={true}
        type="submit"
        variant="contained"
      >
        Sign Up
      </WithStyles(ForwardRef(Button))>
      <WithStyles(ForwardRef(Grid))
        container={true}
        justify="flex-end"
      >
        <WithStyles(ForwardRef(Grid))
          item={true}
        >
          <WithStyles(ForwardRef(Link))
            component={
              Object {
                "$$typeof": Symbol(react.forward_ref),
                "propTypes": Object {
                  "innerRef": [Function],
                  "onClick": [Function],
                  "replace": [Function],
                  "target": [Function],
                  "to": [Function],
                },
                "render": [Function],
              }
            }
            to="/login"
            variant="body2"
          >
            Already have an account? Sign in
          </WithStyles(ForwardRef(Link))>
        </WithStyles(ForwardRef(Grid))>
      </WithStyles(ForwardRef(Grid))>
    </form>
  </div>
  <Styled(MuiBox)
    mt={5}
  >
    <Copyright />
  </Styled(MuiBox)>
</WithStyles(ForwardRef(Container))>
`;
